src/components/Button.js

```diff
@@ -1,19 +1,22 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useContext } from 'react';
 import { StyleSheet } from 'react-native';
 import { Button as RNEButton } from 'react-native-elements';
 
 import { colors, normalize } from '../config';
 import { DiagonalGradient } from './DiagonalGradient';
+import { OrientationContext } from '../OrientationProvider';
 
 export const Button = ({ title, onPress }) => {
+  const { orientation } = useContext(OrientationContext);
   return (
     <RNEButton
       onPress={onPress}
       title={title}
       titleStyle={styles.titleStyle}
       containerStyle={styles.containerStyle}
       ViewComponent={DiagonalGradient}
+      buttonStyle={orientation === 'landscape' ? styles.buttonStyle : {}}
```

Comment:
you could leave it undefined if you want, when the condition does not match. example: `orientation === 'landscape' && styles.buttonStyle`